Update install-snaps.md (Fixes #895) (#901)
MetaMask/metamask-docs: snaps/get-started/install-snaps.md

[MetaMask Flask](https://metamask.io/flask/) is an experimental playground that provides developers 
access to upcoming MetaMask features. It is available as a browser extension.

> While a small set of audited snaps are allowlisted in MetaMask stable, MetaMask Flask is intended for developers building and testing snaps on localhost or from npm. Also, new Snaps API features are enabled in Flask for testing and developer feedback before being enabled in MetaMask stable.

Reviewer: ```suggestion
While a small set of audited snaps are allowlisted in the stable version of the MetaMask browser extension, MetaMask Flask is intended for developers building and testing snaps locally or from npm. Also, new Snaps API features are enabled in Flask for testing and developer feedback before they're enabled in MetaMask stable.
```
